packages/cadl-ranch-specs/package.json

```diff
@@ -47,5 +47,6 @@
     "eslint": "^8.20.0",
     "rimraf": "^3.0.2",
     "typescript": "^4.7.4"
-  }
+  },
+  "peerDependencies": {"@azure-tools/cadl-azure-core": "~0.6.0"}
```

Comment:
I think all of the cadl deps should be moved here instead.

Reply:
ah ok so all cadl deps should be peer dependencies